packages/@lwc/engine-server/src/utils/mutation-tracking.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { HostAttributesKey, HostElement, HostNamespaceKey } from '../types';
+
+const elementsToTrackForMutations: WeakSet<HostElement> = new WeakSet();
+
+const ATTRIBUTE_NAME = 'data-lwc-host-mutated';
+
+export function reportMutation(element: HostElement) {
+    if (elementsToTrackForMutations.has(element)) {
+        const hasAttribute = element[HostAttributesKey].find(
```

Comment:
```suggestion
        const hasMutationAttribute = element[HostAttributesKey].find(
```
Makes it more clear at a glance what kind of attribute we're caring about.